eos/b-decays/b-to-gamma-l-nu.cc

```diff
@@ -0,0 +1,214 @@
+
+        std::tuple<double, double> Gamma_F_Gamma_B(const double & E_gamma_min) const
+        {
+            const double m_B2 = m_B * m_B, m_B3 = m_B2 * m_B;
+
+            auto dGamma_dE_gamma_forward = [&](const double & E_gamma) -> double {
+                const double F_V = form_factors->F_V(E_gamma);
+                const double F_A = form_factors->F_A(E_gamma) + e_l * f_B / E_gamma; // mind different definitions of F_A between [BBJW:2018A] and [BR:2011A]
+
+                return (1. / m_B3 * 1. / power_of<3>(E_gamma - m_B) * (-4 * power_of<2>(E_gamma) * (power_of<2>(F_A) + power_of<2>(F_V)) - 2 * m_B2 * (-3 * F_A * F_V + 2 * power_of<2>(F_A) + 2 * power_of<2>(F_V)) + E_gamma * m_B * (-6 * F_A * F_V + 7 * power_of<2>(F_A) + 7 * power_of<2>(F_V))) * power_of<2>(-2 * E_gamma + m_B) * power_of<3>(E_gamma)) / 3.;
+            };
+
+            auto dGamma_dE_gamma_backward = [&](const double & E_gamma) -> double {
+                const double F_V = form_factors->F_V(E_gamma);
+                const double F_A = form_factors->F_A(E_gamma) + e_l * f_B / E_gamma; // mind different definitions of F_A between [BBJW:2018A] and [BR:2011A]
+
+                return (1. / m_B2 * (2 * E_gamma - m_B) * 1. / power_of<3>(E_gamma - m_B) * (2 * m_B2 * (3 * F_A * F_V + 2 * power_of<2>(F_A) + 2 * power_of<2>(F_V)) + 6 * power_of<2>(E_gamma) * power_of<2>(F_A + F_V) - 9 * E_gamma * m_B * power_of<2>(F_A + F_V)) * power_of<3>(E_gamma)) / 3.;
+            };
+
+            const double prefactor = alpha_qed * power_of<2>(g_fermi * v_ub_abs)
+                    / (16.0 * power_of<2>(M_PI)) * power_of<3>(m_B);
+
+            const double Gamma_forward  = prefactor * integrate<GSL::QAGS>(dGamma_dE_gamma_forward,  E_gamma_min, m_B / 2.0);
+            const double Gamma_backward = prefactor * integrate<GSL::QAGS>(dGamma_dE_gamma_backward, E_gamma_min, m_B / 2.0);
+
+            return {Gamma_forward, Gamma_backward};
+        }
+
+        double A_FB(const double & E_gamma_min) const
+        {
+            const auto [Gamma_forward, Gamma_backward] = Gamma_F_Gamma_B(E_gamma_min);
+
+            return (Gamma_forward - Gamma_backward) / (Gamma_forward + Gamma_backward);
+        }
+    };
+
+    const std::vector<OptionSpecification>
+    Implementation<BToGammaLeptonNeutrino>::options
+    {
+        { "form-factors", { "FLvD2022QCDF" }, "FLvD2022QCDF" }
+    };
+
+    BToGammaLeptonNeutrino::BToGammaLeptonNeutrino(const Parameters & parameters, const Options & options) :
+        PrivateImplementationPattern<BToGammaLeptonNeutrino>(new Implementation<BToGammaLeptonNeutrino>(parameters, options, *this))
+    {
+    }
+
+    BToGammaLeptonNeutrino::~BToGammaLeptonNeutrino()
+    {
+    }
+
+    double
+    BToGammaLeptonNeutrino::branching_ratio(const double & E_gamma_min) const
+    {
+        return _imp->branching_ratio(E_gamma_min);
+    }
+
+    std::tuple<double, double>
+    BToGammaLeptonNeutrino::Gamma_F_Gamma_B(const double & E_gamma_min) const
```

Comment:
Please remove this exported function. If you want to test it, use either the diagnostics or test via A_FB and BR.